### DISCUSSION TITLE
Post install 8.0 + script

### DISCUSSION THREAD

**Discussion by MoldyTaint** [tteck/Proxmox#1529] [Help Needed] [answered]
With a fresh 8.0 install will the post install script be okay? Not sure if it's specific and updated for bookworm or not.

**Comment by tteck** [comment, marked as answer]
The script has been updated to work with either PVE 7 or PVE 8. https://github.com/tteck/Proxmox/commit/9e27e197463e69d577abe2a2560f00f78c22f23e

**Reply by MoldyTaint** [reply]
You are a SAINT! Thank you for all the hard work you do thus far!

**Comment by darthrater78** [comment]
Maybe a silly question but should post install be ran after an upgrade?

**Reply by tteck** [reply]
Not needed

**Comment by zenzip** [comment]
a little bit confused ... for pve 8 which post-install script to use ? 

- post-pve-install.sh
- pve-post-install.sh

**Reply by tteck** [reply]
![Screenshot 2023-06-25 7 35 54 AM](https://github.com/tteck/Proxmox/assets/21342670/0db9afa0-3e1c-4a20-9f75-b039ede1cb8a)

**Reply by zenzip** [reply]
thanks, my concern was if that page was outdated :)

**Reply by tteck** [reply]
`pve-post-install.sh` doesn't exist.

**Reply by zenzip** [reply]
This was confusing me :)
![image](https://github.com/tteck/Proxmox/assets/3242904/020ad5d6-40e6-4348-a725-65913d15d4ca)

**Reply by tteck** [reply]
I think the next commit, they (`post-pve-install.sh` `pve-post-install.sh`) were combined. I'm impressed that you actually investigated!
